### ISSUE TITLE
Web - Screen Reader Does Not Announce if Menu or Sub-menu is Opened or Closed

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#149]
The Menu or sub-menu items do not announce to the screen reader user that the item is now open or closed.

The use of the ARIR-LIVE region is what would be needed. For some reason, my screen reader, JAWS, is telling me that I used the Enter key or the spacebar. Thinking it's because of the ARIA-LIVE region being used. So changing the text might be an easier fix.

**Comment by harpreetsdev** [comment]
@jfc3-dol 

Appreciate you taking the time to review the changes. I have used aria-live attribute in this issue. Let me review the code and get back to you.

@pbhatt17 @Abychalil

**Comment by harpreetsdev** [comment]
@jfc3-dol 

What text do you want the VO to announce for Sub-menu?

@pbhatt17 @Abychalil

**Comment by harpreetsdev** [comment]
@jfc3-dol 

This is what is was announcing previously - screenshots below - 

![Collapsed](https://user-images.githubusercontent.com/13814932/91594355-c40ad380-e92f-11ea-8f68-9b0477d0f987.png)


![Expanded](https://user-images.githubusercontent.com/13814932/91594359-c8cf8780-e92f-11ea-8665-89b882b76555.png)

**Comment by harpreetsdev** [comment]
@jfc3-dol 

I have changed the Expanded and Collapsed to - Opened and Closed.

![Closed](https://user-images.githubusercontent.com/13814932/91594629-31b6ff80-e930-11ea-99c6-c7f722fa4767.png)

![Opened](https://user-images.githubusercontent.com/13814932/91594637-38de0d80-e930-11ea-91c8-3efe5fb876fd.png)


Please let me know if that works, you want some more specific changes.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

I'm not getting those messages when I use JAWS in IE11 and Chrome even after hard-refreshing my browsers. Plus, I rebooted my machine the other day, so I would think that would clear it out.

Let me try again tomorrow after rebooting tonight.

**Comment by jfc3-dol** [comment]
I tested with IE 11, Edge, and Chrome, and the Menu does not tell the assistive technology user (screen reader, etc.) if it is open or closed (expanded or collapsed) like it does for the four show/hide items on things like the main Steps pages.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

I'm having issues with JAWS reading the "Menu" is open or closed because, depending on the browser, I can not get to it. I'm also getting cases where the focus is on a given link, but JAWS is reading other text from a previous part of the page. It's like the focus for JAWS is not where the visual focus is being set.

Sometimes JAWS is skipping over the main content information and only reading the header and footer parts of the page in IE 11.

For these issues, I attempted to use JAWS on another application I was using, and things seem to be working as I would expect them and not skipping parts of the page or misreading things using IE 11, Chrome, and Edge.

So I'm thinking it more the application than the screen reader and browser combinations.

I rebooted my machine and still have the issue. So I need to reboot it again, maybe tomorrow, to see if that fixes things or not.

**Comment by jfc3-dol** [comment]
@pbhatt17 

JAWS is only reading the Footer information for IE 11, Edge, and Chrome. Not sure why when I tab back to other links on the page and attempt to read them, and JAWS will not read it and moves to the footer "Bookmarks" link.

So I'm leaving this issue open since I can't get JAWS to read anything else and the "Menu' button to tell if it is open or closed.

**Comment by jfc3-dol** [comment]
@pbhatt17 

My JAWS is only reading the footer items and I can't anything else.
